### PR TITLE
Add Mises Network

### PR DESCRIPTION
**Pull request by OnepercentTech**

### CODE REVIEW

File: mises/chain.json [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+  "website": "https://mises.site",
+  "pretty_name": "Mises",
+  "chain_id": "mainnet",
+  "bech32_prefix": "mis",
+  "node_home": "$HOME/.misestm",
+  "daemon_name": "misestmd",
+  "key_algos": [
+    "secp256k1"
+  ],
+  "slip44": [60, 118],
```
Comment (JeremyParish69) [inline mises/chain.json:15]: the type supports only a number. Are both coinTypes are supported?

File: mises/chain.json [hunk 2]
```diff
@@ -0,0 +1,127 @@
+{
+  "$schema": "../chain.schema.json",
+  "chain_name": "mises",
+  "status": "live",
+  "network_type": "mainnet",
+  "website": "https://mises.site",
+  "pretty_name": "Mises",
+  "chain_id": "mainnet",
+  "bech32_prefix": "mis",
```
Comment (JeremyParish69) [inline mises/chain.json:9]: Please register this prefix
https://github.com/satoshilabs/slips/blob/master/slip-0173.md

File: mises/chain.json (hunk 1 above)
Comment (jambulmerah) [reply mises/chain.json:15]: Yes, both are supported

File: mises/chain.json (hunk 2 above)
Comment (jambulmerah) [reply mises/chain.json]: Registered and fixed typo from `mis` to `mises`
https://github.com/satoshilabs/slips/commit/1cbb46e974d306d512f16dbe6cfa23dffa86990b
https://github.com/cosmos/chain-registry/pull/1068/commits/94a6b5ca29dacaa998da5aa611f3342e97a3e7e9

**Pull request closed without merging**